src/main/java/devcourse/board/api/controller/PostApiController.java

```diff
@@ -25,11 +29,11 @@ public ResponseEntity<PostResponse> getPost(
             @PathVariable Long postId
     ) {
         return ResponseEntity.ok()
-                .body(postService.findOne(postId));
+                .body(postService.getPost(postId));
     }
 
     @GetMapping
-    public ResponseEntity<MultiplePostResponse> getPosts(
+    public ResponseEntity<MultipleSimplePostResponse> getPosts(
```

Comment:
```suggestion
    public ResponseEntity<PostResponses> getPosts(
```
복수로 표현하면 짧고 명료하게 표현할 수 있지않을까요?

Reply:
나중에 코드를 작성할 때,
`PostResponses postResponses = ...`
`List<PostResponse> postResponses = ... `
처럼 로컬변수명이 겹칠 수도 있을 것 같아 일부러 클래스이름을 복수형으로 작성하지 않았었는데요, 
지금 생각해보니 위 두 코드가 하나의 메서드에서 작성될 일은 거의 없을 것 같습니다. 불필요하게 나중 상황을 고려했던 것 같아요.🥲 
감사합니다!